File: typescript/array/RichestCustomerWealth.ts
/**
 * LeetCode 1672. Richest Customer Wealth
 * https://leetcode.com/problems/richest-customer-wealth/
 */
function maximumWealth(accounts: number[][]): number {
    let max: number = 0;
    accounts.forEach((v: number[], i: number) => {
        let val: number = 0;
        v.forEach((n: number, j: number) => {
            val += n;
        });
        if (val > max) {
            max = val;
        }
    });
    return max;
};

// EOF

File: go/array/RichestCustomerWealth.go
package main

/**
 * LeetCode 1672. Richest Customer Wealth
 * https://leetcode.com/problems/richest-customer-wealth/
 */
func maximumWealth(accounts [][]int) int {
	max := 0
	for i := range accounts {
		val := 0
		for j := range accounts[i] {
			val += accounts[i][j]
		}
		if val > max {
			max = val
		}
	}
	return max
}

func main() {
	// ...
}

// EOF
